sprint6/DO_NOT_TOUCH/zadanie2.js

```diff
@@ -0,0 +1,18 @@
+/*
+Zadanie 2. Stwórz funkcję, która przyjmie tablicę liczb całkowitych i zwróci ich sumę.
+*/
+
+function sumNumber(arr) {
```

Comment:
Nazwa funkcji powinna odzwierciedlać to co robi funkcja. Czyli w tym przypadku np. `zwróćSumęLiczbZTablicy`